pkg/settings/setting_test.go

```diff
@@ -3,6 +3,9 @@ package settings
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/sirupsen/logrus"
```

Comment:
We try to separate imports from the standard library and other imports with a newline, 
i.e.
```go
import (
	"encoding/json"
	"fmt"	
	"io"
	"os"
	"testing"

	v1 "k8s.io/api/core/v1"
        "github.com/sirupsen/logrus"
)
```
This is a pattern we try and follow throughout Rancher

Reply:
Done.  
Is there a reason why you putted the v1 over logrus?

Reply:
No, I just wrote the code block quickly and didn't sort the names properly